src/rules/no-useless-path-segments.js

```diff
@@ -39,6 +39,8 @@ module.exports = {
       url: docsUrl('no-useless-path-segments'),
     },
 
+    schema: [ makeOptionsSchema() ],
```

Comment:
this isn’t defining “commonjs” as an option tho

Reply:
doesn't line 111 from this function define it?
https://github.com/benmosher/eslint-plugin-import/blob/37554fe9844986a1ab126d2a8f59fe4c7551a2f8/utils/moduleVisitor.js#L107-L122

Reply:
oh, hmm. I don’t think it’s a good idea to add a bunch of options that aren’t being used. Can we make a schema that only has the commonjs option?

Reply:
from what i observe, every option is valid, function `makeOptionsSchema` is made specifically for rule that uses util `moduleVisitor`
checkout
https://github.com/benmosher/eslint-plugin-import/blob/37554fe9844986a1ab126d2a8f59fe4c7551a2f8/utils/moduleVisitor.js#L13-L101

but it's your call, i can add only commonjs

Reply:
it’s fine to add support for every option, but then let’s make sure to add tests for all the options.